source/uwp/Renderer/lib/XamlBuilder.cpp

```diff
@@ -623,6 +631,7 @@ namespace AdaptiveCards { namespace XamlCardRenderer
 
         UINT currentAction = 0;
 
+        ComPtr<XamlCardRenderer> strongRenderer(renderer);
```

Comment:
Chris had asked that i call it "strong" rather than "local" on a previous change where i had defined the com pointer for the purpose of taking a strong reference to pass to a lambda.  

---
In reply to: [116594514](https://github.com/Microsoft/AdaptiveCards/pull/329#discussion_r116594514) [](ancestors = 116594514)